src/pybind/mgr/dashboard/frontend/src/app/shared/services/scroll-restoration.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { Router, NavigationEnd } from '@angular/router';
+import { filter } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ScrollRestorationService {
+    constructor(private router: Router) {
+        this.router.events
+            .pipe(filter((event) => event instanceof NavigationEnd))
```

Comment:
wont this mean for every insatnce of NavigationEnd it will capture and scroll to top even if I am just scrolling page down ?